screens/auth/LogInScreen.js

```diff
@@ -88,6 +78,13 @@ export default class LogInScreen extends React.Component {
       // Returns empty array if no customer is found
       if (customers.length === 1) {
         const customer = customers[0];
+        // register this user in the Sentry logger
+        Sentry.configureScope(scope => {
+          scope.setUser({
+            id: customer.id,
```

Comment:
I would guess this is a privacy/security concern

Reply:
@wangannie @anniero98 
For any sensitive data -- sentry will automatically scrub the data and those options can be modified on the dashboard -- which affects how it appears in logs -- passwords for example will automatically be redacted. username should be fine -- I think i would be more inclined to to not log the phone number if it's not necessary, since that's information that is personally identifiable outside of the context of the application, vs the id and username is DC Central app generated content that identifies a user, which can still be used to effectively debug.

 if it's not necessary to log the phone number, you can if you can just log the `id` + `username` both should be enough to effectively debug but those are my initial concerns